composer.json

```diff
@@ -34,7 +34,7 @@
         "drupal/facets": "^1.0",
         "drupal/features": "^3.5",
         "drupal/field_group": "^3.0",
-        "drupal/geysir": "^1.1",
+        "drupal/geysir": "^1.x-dev",
```

Comment:
Nie możemy tak robić, bo chcemy być stabilni - jeśli to kwestia bugów geysira to wyłączmy preview na razie, przywrócimy go jak wyjdzie nowa wersja.